python/daqconf/apps/trigger_gen.py

```diff
@@ -282,14 +282,16 @@ def get_trigger_app(
                               conf = tcm.Conf(candidate_maker=CANDIDATE_PLUGIN[j],
                                      candidate_maker_config=temptypes.CandidateConf(CANDIDATE_CONFIG[j]))),
 
-                        DAQModule(name = f'tctee_chain_{j}',
+                        DAQModule(name = f'tctee_swt_{j}',
                               plugin = 'TCTee'),]
 
-        for tp_sid in TP_SOURCE_IDS:
-            link_id = f'tplink{tp_sid}'
+        for tp_sid,tp_conf in TP_SOURCE_IDS.items():
+            ru_sid = f'{tp_conf.tp_ru_sid}'
+            region = f'{tp_conf.region_id}'
+            plane = f'{tp_conf.plane}'
 
             if USE_CHANNEL_FILTER:
-                modules += [DAQModule(name = f'channelfilter_{link_id}',
+                modules += [DAQModule(name = f'tpcf_{region}_{plane}',
```

Comment:
Not a code issue, just an idea that we should perhaps make a legend somewhere (github doc?) for the different acronyms for our modules / thread names (specifically tpcf - while somewhat obvious to us - may not be immediately recognised).